### PR TITLE
update healthbar

### PR DESCRIPTION
**Pull request by OwenBoisse**
move healthbar to above boss
change to 3d oblong
make healthbar creation a reusable function
move health component to health_bar.rs

### CODE REVIEW

File: src/health_bar.rs [hunk 1]
```diff
@@ -60,54 +64,68 @@ fn spawn_health_bar(
...
-            Without<HealthBar>,
+            With<HealthBar>,
+            Without<Boss>,
+            Without<Player>,
         ),
     >,
 ) {
-    let left_eye = left_eye.get_single().unwrap();
-    let right_eye = right_eye.get_single().unwrap();
+    if let Some(player_transform) = player_query.iter().next() {
```
Comment (rosefromthedead) [inline src/health_bar.rs:88]: could this be a let-else? and `query.iter().next()` can be `query.get_single()`

**Review by rosefromthedead — APPROVED** [review]
Assuming this has been tested to look cool, send it, thanks!

**Comment by OwenBoisse** [comment]
currently changing healthbar system further, dont approve this pr yet

**Comment by OwenBoisse** [comment]
done i think

**Review by rosefromthedead — APPROVED** [review]
Honestly too frazzled to comprehend this but I trust you've tested it and it looks pretty harmless

**Review by a6-webm — APPROVED** [review]
Approving because I need to hop on master lol

**Comment by a6-webm** [comment]
Ah yeah I think this PR is possibly defunct now, because I had to cherry pick the healthbar update into [boss_state_machine](https://github.com/goudanough/WizARds/tree/boss_state_machine).

**Review by Foxinatel — CHANGES_REQUESTED** [review]
Going to place a block on this PR since it has been reported as potentially defunct

**Comment by rosefromthedead** [comment]
Closing in favour of merging it with the rest of boss_state_machine

**Pull request closed without merging**